src/Desktop/UIAutomation/CustomObjects/PropertyDescription.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Axe.Windows.Desktop.UIAutomation.CustomObjects
+{
+    public class PropertyDescription
+    {
+        public Guid Id { get; }
+        public string Name { get; }
```

Comment:
Do we want to be consistent here with the json names?
```suggestion
        public string programmaticName { get; }
```